src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -1528,6 +1529,9 @@ def _is_valid_uri(uri):
 def _add_sas_for_url(cmd, url, account_name, account_key, sas_token, service, resource_types, permissions):
     from azure.cli.command_modules.storage.azcopy.util import _generate_sas_token
 
+    if '&sig=' in url:
+        return url
```

Comment:
do you want to check if it is sas url or not?
If yes, prefer utilizing `_is_valid_uri()` in L1518.

Reply:
Thanks! Modified.